shared/actions/wallets.js

```diff
@@ -205,7 +206,7 @@ function* requestPayment(state) {
   )
   const navAction = maybeNavigateAwayFromSendForm(state)
   yield Saga.sequentially([
-    ...(navAction ? [Saga.put(navAction)] : []),
```

Comment:
maybeNavigateAwayFromSendForm always returns an array that needs to be mapped now